src/egon/data/datasets/industrial_gas_demand.py

```diff
@@ -460,104 +472,165 @@ def insert_industrial_gas_demand_egon100RE():
     None
 
     """
+    if "eGon100RE" not in config.settings()["egon-data"]["--scenarios"]:
+        logger.warning(
+            """
+        Scenario eGon100RE not in list of required scenarios. Taks skipped.
+        """
+        )
+        return
+
     scn_name = "eGon100RE"
     delete_old_entries(scn_name)
 
-    # read demands
-    industrial_gas_demand_CH4 = read_and_process_demand(
-        scn_name=scn_name, carrier="CH4_for_industry", grid_carrier="CH4"
+    # Deal with egon_etrago_load
+    load_gas = pd.read_csv(
+        "data_bundle_powerd_data/egon_etrago_load_gas.csv",
     )
-    industrial_gas_demand_H2 = read_and_process_demand(
-        scn_name=scn_name, carrier="H2_for_industry", grid_carrier="H2_grid"
+    load_gas.rename(columns={"load_id": "load_id_orig"}, inplace=True)
+    load_gas = load_gas[load_gas["scn_name"] == scn_name]
+    load_gas.drop(columns=["bus"], inplace=True)
+    load_gas.set_index("load_id_orig", inplace=True)
+
+    load_gas["geom"] = gpd.GeoSeries.from_wkt(load_gas["geom"])
+    load_gas = gpd.GeoDataFrame(
+        load_gas,
+        geometry="geom",
+        crs=4326,
     )
+    loads_h2 = load_gas[load_gas["carrier"] == "H2_for_industry"]
+    loads_ch4 = load_gas[load_gas["carrier"] == "CH4_for_industry"]
+    loads_h2 = db.assign_gas_bus_id(loads_h2, "eGon100RE", "H2_grid")
+    loads_ch4 = db.assign_gas_bus_id(loads_ch4, "eGon100RE", "CH4")
 
-    # adjust H2 and CH4 total demands (values from PES)
-    # CH4 demand = 0 in 100RE, therefore scale H2 ts
-    # fallback values see https://github.com/openego/eGon-data/issues/626
-    n = read_network()

```

Comment:
The data from pypsaeur is still accessible, isn't it? Or was there another reason for dropping these lines?